### PR TITLE
#49 docker instead solr wrapper

### PR DESCRIPTION
**Pull request by cdmo**
1. changes namespace to solr from psulib_traject:solr
2. adds 3 tasks, up, clean and stop
3. updates documentation accordingly

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -88,9 +74,8 @@ DEPENDENCIES
   marc
   rake
   rsolr
-  solr_wrapper!
   traject (= 3.0.0.alpha.2)
```
Comment (banukutlu) [inline Gemfile.lock]: traject 3.0 did not work?

**Comment by banukutlu** [comment]
```bundle exec rake solr:up
felix
rake aborted!
Don't know how to build task 'docker:ps' (see --tasks)
```

need this task: 

```
  task :ps do
    print `docker ps`
  end
```

**Comment by banukutlu** [comment]
I added `require 'faraday'` for solr:conf to work and removed the obsolete solr conf files from the list of files to be downloaded

File: Gemfile.lock (hunk 1 above)
Comment (banukutlu) [reply Gemfile.lock]: with 1f463ce18 this is solved

Comment (cdmo) [reply Gemfile.lock]: hmm thought i did a bundle install... maybe i need to specifically do bundle update?

Comment (cdmo) [reply Gemfile.lock]: so that is outdated, i think i caught this mistake after-the-fact

**Comment by cdmo** [comment]
I am getting that too. Not sure why it's returning a 400 *and* successfully clearing the index. Maybe docker something something.

**Pull request merged**